src/benchmarks/gc/docs/bench_file.md

```diff
@@ -58,7 +61,7 @@ benchmarks:
 comment: `str | None`
   (ignored)
 
-vary: `"machine" | "executable" | "coreclr" | "config" | "benchmark" | None`
+vary: `"machine" | "coreclr" | "config" | "benchmark" | "executable" | None`
```

Comment:
not sure why the order was changed :P

Reply:
The file is automatically generated by `py . lint`. Presumably, it wants the options to be alphabetically ordered.

Reply:
but it's not alphabetically ordered 😄... I can't tell what order this is. @ivdiazsa if you run the linter does it generate the same order for you? if not we may want to disable this from the linter (also I'm surprised since I thought you ran the linter on the whole code base).

Reply:
Yes, it also happened to me. I don't know how the linter determines which order to use for these values. Since it looked odd to me, I manually undid this change in my PR's.

Reply:
Let's keep it the way the linter likes so we don't have to revert it all the time? I don't know how the linter determines that order either.